plugins/azure/virtualmachines/oldVmDiskSnapshots.js

```diff
@@ -0,0 +1,67 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Old VM Disk Snapshots',
+    category: 'Virtual Machines',
+    description: 'Ensures that virtual machines do not have older disk snapshots.',
+    more_info: 'A snapshot is a full, read-only copy of a virtual hard drive (VHD). You can take a snapshot of an OS or data disk VHD to use as a backup, or to troubleshoot virtual machine (VM) issues. VM snapshots older than a specific period of time should be deleted to save cost of unused resources.',
+    recommended_action: 'Ensure that there are no undesired old VM disk snapshots',
+    link: 'https://docs.microsoft.com/en-us/azure/virtual-machines/windows/snapshot-copy-managed-disk',
+    apis: ['snapshots:list'],
+    settings: {
+        Days_Since_Snapshot_Creation: {
+            name: 'Days since the creation of snapshot',
+            description: 'The number of days since snapshot was created',
+            regex: '^[0-9]*',
+            default: '30'
```

Comment:
```suggestion
        days_since_snapshot_creation: {
            name: 'Days Since Snapshot Creation',
            description: 'The number of days since snapshot was created. Snapshots older than this value of days should be deleted',
            regex: '^[0-9]*',
            default: '30'
```